README.md

```diff
@@ -44,12 +44,34 @@ cargo build --release
 
 **Pura** can be used as a standalone container runtime like **runc** with the OCI compliant commands:
 
+Make this to clean the way:
+```sh
+set ID=example
+[sudo] unlink /tmp/pura/${ID}/init.sock
+[sudo] unlink /tmp/pura/${ID}/container.sock
```

Comment:
This shouldn't be in the README as it's a bug in the runtime with the cleanup. You can make a new file `NOTES.md` for internal contributor notes and add these comments. Same for the `netcat` command below.